src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -130,9 +135,13 @@ public partial struct JsonDocumentOptions
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
+        public bool IsImmutable { get { throw null; } }
         public System.Text.Json.JsonElement this[int index] { get { throw null; } }
         public System.Text.Json.JsonValueKind ValueKind { get { throw null; } }
         public System.Text.Json.JsonElement Clone() { throw null; }
+        public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonDocument jsonDocument) { throw null; }
+        public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonElement jsonElement) { throw null; }
+        public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonNode jsonNode) { throw null; }
```

Comment:
Why does JsonElement have a static method that takes and produces a JsonNode?  These DeepCopy methods seem like they belong on JsonNode, not JsonElement.

Reply:
Yes, I'll move it to JsonNode.